resources/tickets.py

```diff
@@ -70,11 +71,8 @@ def get(self):
 
     @pm_level_required
     def post(self):
-        data = Tickets.parser.parse_args()
-        ticket = TicketModel(**data)
-
-        ticket.save_to_db()
-        return ticket.json(), 201
+        TicketModel.create(schema=TicketSchema, payload=request.json)
+        return {"message": "Ticket successfully created"}, 201
```

Comment:
We are not using the returned JSON of the created ticket at all in the frontend (PR: https://github.com/codeforpdx/dwellingly-app/pull/611) so I changed this to just return a success message